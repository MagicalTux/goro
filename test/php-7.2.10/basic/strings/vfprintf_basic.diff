     *** Testing vfprintf() : basic functionality ***
002- string(17) "Foo is 30 and bar"
003- int(17)
004- string(11) "bar bar bar"
005- int(11)
006- string(8) "54 digit"
007- int(8)
008- string(3) "1 0"
009- int(3)
010- string(5) "A B C"
011- int(5)
012- string(35) "1.000000e+3 2.000000E+4 2.000000e+2"
013- int(35)
014- string(2) "50"
015- int(2)
016- string(35) "Testing 1001 6 2.550200 foobar f 41"
017- int(35)
018- ===DONE===
002+ Warning: fopen(): fopen(vfprintf_test.txt): failed to open stream: No such file or directory in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 22
003+ 
004+ Fatal error: Uncaught Error: Call to undefined function ftruncate in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php:10
005+ Stack trace:
006+ #0 {main}
007+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 10
