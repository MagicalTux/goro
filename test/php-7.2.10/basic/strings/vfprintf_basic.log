
---- EXPECTED OUTPUT
*** Testing vfprintf() : basic functionality ***
string(17) "Foo is 30 and bar"
int(17)
string(11) "bar bar bar"
int(11)
string(8) "54 digit"
int(8)
string(3) "1 0"
int(3)
string(5) "A B C"
int(5)
string(35) "1.000000e+3 2.000000E+4 2.000000e+2"
int(35)
string(2) "50"
int(2)
string(35) "Testing 1001 6 2.550200 foobar f 41"
int(35)
===DONE===
---- ACTUAL OUTPUT
*** Testing vfprintf() : basic functionality ***
Warning: fopen(): fopen(vfprintf_test.txt): failed to open stream: No such file or directory in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 22

Fatal error: Uncaught Error: Call to undefined function ftruncate in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php:10
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 10
---- FAILED
