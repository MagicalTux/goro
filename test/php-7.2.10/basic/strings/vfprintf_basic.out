*** Testing vfprintf() : basic functionality ***
Warning: fopen(): fopen(vfprintf_test.txt): failed to open stream: No such file or directory in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 22

Fatal error: Uncaught Error: Call to undefined function ftruncate in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php:10
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/vfprintf_basic.php on line 10